client/src/components/layout/TopBar.js

```diff
@@ -20,6 +20,15 @@ const TopBar = ({ user }) => {
     </li>,
   ];
 
+  let newPostLink
+  if (user) {
+    newPostLink = (
+      <li className="menu-text">
+        <Link to="/salads/new">Post a salad!</Link>
+      </li>
```

Comment:
this doesnt need to be an li. delete the li. If you need the styling, then you can add a div, or apply the styling as a className directly to the Link